zschema/tests.py

```diff
@@ -615,10 +667,10 @@ def test_get_registered(self):
 class SubRecordTests(unittest.TestCase):
 
     def test_subrecord_child_types_can_override_parent_attributes(self):
-        Certificate = SubRecord({}, doc="A parsed certificate.")
+        Certificate = SubRecordType({}, doc="A parsed certificate.")
         OtherType = SubRecord({
-            "ca": Certificate.new(doc="The CA certificate."),
-            "host": Certificate.new(doc="The host certificate."),
+            "ca": Certificate(doc="The CA certificate."),
+            "host": Certificate(doc="The host certificate."),
         })
         self.assertEqual("A parsed certificate." , Certificate.doc)
```

Comment:
To be identical with the old test, it should actually be ` self.assertEqual("A parsed certificate." , Certificate().doc)`. The current behavior is the same check as done on line 619.